google/cloud/spanner_dbapi/parser.py

```diff
@@ -138,18 +146,33 @@ def all_have_same_argc(self):
 
 
 class values(a_args):
-    def __str__(self):
-        return "VALUES%s" % super().__str__()
+    """Wrapper for values.
 
+    :rtype: str
+    :returns: A string of values expression in tree view.
+    """
 
-def parse_values(stmt):
-    return expect(stmt, VALUES)
+    def __str__(self):
+        return "VALUES%s" % super().__str__()
```

Comment:
Have you erased the `parse_values` func?

Reply:
Strange thing. <img width="661" alt="Screenshot 2020-10-06 at 14 58 01" src="https://user-images.githubusercontent.com/27859234/95198721-57171300-07e4-11eb-837c-01c5f3b1c56a.png">

Reply:
So that's not my change.